integrations/terraform-cloud/.port/resources/blueprints.json

```diff
@@ -123,9 +123,42 @@
             "target":"terraformCloudStateVersion",
             "required":false,
             "many":false
-         }
+         },
+         "project":{"title":"Terraform Cloud Project",
+          "target":"terraformCloudProject",
+          "required":false,
+          "many":false}
       }
    },
+   {
+      "identifier": "terraformCloudProject",
+      "description": "This blueprint represents a project in Terraform Cloud",
+      "title": "Terraform Cloud Project",
+      "icon": "Terraform",
+      "schema": {
+        "properties": {
+          "name": {
+            "type": "string",
+            "title": "Project Name",
+            "description": "The name of the Terraform Cloud project"
+          },
+          "permissions": {
+            "type": "object",
+            "title": "Permissions",
+            "description": "The permisssions on the project"
+          },
+          "organizationId": {

```

Comment:
change the order to be in the order of relations for cleaner reading of the blueprints json.

project > workspace > cloudStateversion > cloudRun